client/blocks/signup-form/passwordless.jsx

```diff
@@ -153,11 +158,8 @@ class PasswordlessSignupForm extends Component {
 					</>
 				);
 			default:
-				return (
-					errorObj.message ||
```

Comment:
I was going to question why you removed the default of showing the error message from the server. But after taking a look at the code I see that the error messages returned by the server aren't exactly user friendly.

Reply:
Yep, that's the only reason I removed it!